### PR TITLE
Provide a Null option in column menu

### PR DESCRIPTION
**Pull request by dgboss**
Adds a 'Null' option to the column menu that sets all editable forecasted values in the column to Null - N/A - NaN.

The option is available for all forecast columns, not just Wind Direction.

# Test Links:
[Landing Page](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2809.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

**Review by andrea-williams — APPROVED** [review]
My only feedback is to reconsider wording of the error message when trying to save an invalid forecast. It still says that a forecast cannot contain any N/A values - maybe reword that to clarify that wind direction can be null.

**Comment by dgboss** [comment]
> My only feedback is to reconsider wording of the error message when trying to save an invalid forecast. It still says that a forecast cannot contain any N/A values - maybe reword that to clarify that wind direction can be null.

Good catch. How about 'Forecast not submitted. A forecast can only contain N/A values for the Wind Direction.'

**Comment by andrea-williams** [comment]
> > My only feedback is to reconsider wording of the error message when trying to save an invalid forecast. It still says that a forecast cannot contain any N/A values - maybe reword that to clarify that wind direction can be null.
> 
> Good catch. How about 'Forecast not submitted. A forecast can only contain N/A values for the Wind Direction.'

Sounds good to me!
